infra/integration-image/Dockerfile.passrate

```diff
@@ -0,0 +1,23 @@
+# Building:
+# docker build --no-cache -t vic-pass-rate -f Dockerfile.passrate ../..
+# docker tag vic-pass-rate gcr.io/eminent-nation-87317/vic-pass-rate:1.x
+# gcloud auth login
+# gcloud docker -- push gcr.io/eminent-nation-87317/vic-pass-rate:1.x
+# download and install harbor certs, then docker login, then:
+# docker tag vic-pass-rate wdc-harbor-ci.eng.vmware.com/default-project/vic-pass-rate:1.x
+# docker push wdc-harbor-ci.eng.vmware.com/default-project/vic-pass-rate:1.x
+
+FROM vmware/photon:2.0
+
+RUN set -eux; \
+     tdnf distro-sync --refresh -y; \
+     tdnf remove toybox -y; \
```

Comment:
If there was a package conflict issue with toybox then let #photon know. They've been pretty good about addressing them.

Reply:
Perhaps I should document this in the Dockerfile itself, but: `toybox` is removed in order to install GNU date (via `coreutils`). I think everything is working as intended from Photon's point of view.